Automattic/simplenote-android
# Add Style Setting (Part 2)

### Fix
This is the second part in a series of pull requests to add a style setting to the app.  The pull requests are split into parts in an attempt to make the changes smaller.  This part adds the ***Black*** style.  See the screenshots below for illustration.

![add_style_setting_all_02](https://user-images.githubusercontent.com/3827611/89597276-de60fe00-d816-11ea-8593-44bdae3e676e.png)

#### Note
Contact me for an installable build.

### Test
1. Open navigation drawer.
2. Tap ***Settings*** item in drawer.
3. Tap ***Style*** item under ***Appearance*** section.
4. Notice ***Style*** screen with ***Default***, ***Black***, and ***Classic*** styles.
5. Tap ***Black*** style.
6. Notice ***Black*** style is selected and app has ***Black*** styling.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.  `RELEASE-NOTES.txt` will be updated once the styles feature is complete.
